common/flatpak-utils.c

```diff
@@ -8057,16 +8057,16 @@ flatpak_get_cursor_pos (int * row, int *col)
   return res == 2;
 }
 
-void
+ssize_t
 flatpak_hide_cursor (void)
 {
-  write (STDOUT_FILENO, FLATPAK_ANSI_HIDE_CURSOR, strlen (FLATPAK_ANSI_HIDE_CURSOR));
+  return write (STDOUT_FILENO, FLATPAK_ANSI_HIDE_CURSOR, strlen (FLATPAK_ANSI_HIDE_CURSOR));
 }
 
-void
+ssize_t
 flatpak_show_cursor (void)
 {
-  write (STDOUT_FILENO, FLATPAK_ANSI_SHOW_CURSOR, strlen (FLATPAK_ANSI_SHOW_CURSOR));
+  return write (STDOUT_FILENO, FLATPAK_ANSI_SHOW_CURSOR, strlen (FLATPAK_ANSI_SHOW_CURSOR));
 }
```

Comment:
This just moves the supression of the error to all the callers. It would be better to have a local error check and a g_warning() on failure.

Reply:
I changed it back to having a `void` return type. This is internal API so we can change it if we want to, but none of the callers need a return value.